client/allocrunner/taskrunner/logmon_hook_test.go

```diff
@@ -1,7 +1,177 @@
+// +build !windows
+
```

Comment:
We should probably have some tests that can run against windows

Reply:
Good call, splitting into unix-specific and portable test files.